rai_test_utils/setup.py

```diff
@@ -15,6 +15,13 @@
 with open('requirements.txt') as f:
     install_requires = [line.strip() for line in f]
 
+extras_require = {
+    "object_detection": [
+        "torch==1.13.1",
+        "torchvision==0.14.1",
+    ]
+}
+
```

Comment:
Thanks for creating the extras.

Reply:
do you mind moving them in requirements-object-detection.txt in this package and then reading from it.